MapboxSpeech.xcodeproj/project.pbxproj

```diff
@@ -70,10 +68,8 @@
 		C5569E6C1FD9EEE100D9B7AC /* hello.mp3 */ = {isa = PBXFileReference; lastKnownFileType = audio.mp3; path = hello.mp3; sourceTree = "<group>"; };
 		DAE3527322420CE900A3FF9C /* MapboxSpeech.framework */ = {isa = PBXFileReference; explicitFileType = wrapper.framework; includeInIndex = 0; path = MapboxSpeech.framework; sourceTree = BUILT_PRODUCTS_DIR; };
 		DAE3527B22420CEA00A3FF9C /* MapboxSpeechTests.xctest */ = {isa = PBXFileReference; explicitFileType = wrapper.cfbundle; includeInIndex = 0; path = MapboxSpeechTests.xctest; sourceTree = BUILT_PRODUCTS_DIR; };
-		DAE3528F22420FDE00A3FF9C /* OHHTTPStubs.framework */ = {isa = PBXFileReference; lastKnownFileType = wrapper.framework; name = OHHTTPStubs.framework; path = Carthage/Build/Mac/OHHTTPStubs.framework; sourceTree = "<group>"; };
```

Comment:
Looks like this framework dependency was removed for all platforms except iOS. Is this because the tests are only being run on iOS?

Reply:
Turns out I was running `$ carthage update --platform iOS`, that's the reason `OHHTTPStubs.framework` were shown as missing ones for other platforms. Interesting part is that even with frameworks being removed from project CircleCI job was passing (and tests for macOS and tvOS were executed). I've added back `OHHTTPStubs.framework` for macOS and tvOS.